src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -555,7 +555,16 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode)
                             ->prototype('scalar')->end()
                             ->defaultValue(array('en'))
                         ->end()
-                        ->booleanNode('logging')->defaultValue($this->debug)->end()
+                        ->arrayNode('logging')
+                            ->canBeEnabled()
+                            ->children()
+                                ->booleanNode('enabled')->defaultValue($this->debug)->end()
+                                ->arrayNode('exclude_domains')
```

Comment:
This is missing the ``fixXmlConfig`` in the parent to support XML properly.

And I suggest using ``excluded_domains`` instead